### ISSUE TITLE
Error: ENOENT: no such file or directory, open 'C:\ae\iconify-react/dist/icon.js'

### ISSUE THREAD

**Issue by peterennis** [iconify-design/iconify-react#6]
Build fail.
Path handling issue with Windows 10.

```
C:\ae\iconify-react>npm i
audited 1398 packages in 2.981s
found 0 vulnerabilities


C:\ae\iconify-react>npm run build

> @iconify/react@1.1.0 build C:\ae\iconify-react
> node build

fs.js:114
    throw err;
    ^

Error: ENOENT: no such file or directory, open 'C:\ae\iconify-react/dist/icon.js'
    at Object.openSync (fs.js:438:3)
    at Object.writeFileSync (fs.js:1189:35)
    at forEach.filename (C:\ae\iconify-react\build.js:29:12)
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (C:\ae\iconify-react\build.js:20:10)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @iconify/react@1.1.0 build: `node build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @iconify/react@1.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\peter\AppData\Roaming\npm-cache\_logs\2019-07-15T04_54_03_225Z-debug.log

C:\ae\iconify-react>
```

**Comment by peterennis** [comment]
https://github.com/peterennis/iconify-react - cloned from latest https://github.com/iconify/iconify-react/commit/b90719647591a52aa83f7eef5f71e4a7e4ac269a

**Comment by cyberalien** [comment]
Thanks for noticing! Forgot to add mkdir to create `dist` in build script. Fixed in version 1.1.1
